api/v1beta1/module_webhook.go

```diff
@@ -55,9 +55,21 @@ func (m *Module) ValidateCreate() (admission.Warnings, error) {
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
-func (m *Module) ValidateUpdate(_ runtime.Object) (admission.Warnings, error) {
+func (m *Module) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
 	modulelog.Info("Validating Module update", "name", m.Name, "namespace", m.Namespace)
 
+	if old != nil {
```

Comment:
can old be nil in the ValidateUpdate? It will mean that the old object does not exists, and in that case ValidateCreate should have been called, no?

Reply:
I agree that it should never be nil. However I have not dug into the implementation and I would not fail if `old` is nil but there is a legitimate reason that we have not thought about.

Reply:
ok, fair enough.